app/services/GuidanceService.scala

```diff
@@ -165,6 +165,7 @@ class GuidanceService @Inject() (
         logger.warn(s"Unable to find process using identifier $processIdentifier, received $err")
         Future.successful(Left(err))
       case Right(process) =>
+        logger.warn(s"Process ${process.meta.id} loaded from backend, containing ${process.flow.keys.toList.length} stanzas and ${process.phrases.length} phrases")
```

Comment:
This isn't really reporting a problem. Probably would be better to use "info" rather than "warn".

Reply:
did this so hopefully we'll see this in prod when a process is first loaded